template/composer.json

```diff
@@ -23,6 +23,15 @@
         "phpunit/phpunit": "^6.3",
         "zendframework/zend-coding-standard": "~1.0.0"
     },
+    "provide": {
+        "psr/package": "<version>"
+    },
+    "conflict": {
+        "conflicted/package": "<version>"
+    },
+    "suggest": {
+        "suggested/package": "<suggested version> <description>"
+    },
```

Comment:
I don't think these should be in the template, as they are (1) optional, and (2) will vary greatly for each package. They can be added if/when they are needed.

Reply:
I've added it because I'm using that template in the script to sort sections always in the same order. We can set sections order in the script and then we don't need to add it here. What do you think?

Reply:
Do it in the script, and not the `composer.json`, please; that way, we don't need to manually update the `composer.json` immediately for new components to make it work. WIth these entries, you'll immediately get failures if you run `composer validate` otherwise, even after making substitutions of `{var}` entries.